lib/taxonomy/level_one_taxons_fetcher.rb

```diff
@@ -0,0 +1,13 @@
+class Taxonomy::LevelOneTaxonsFetcher
+  def self.fetch
+    root_taxon = Whitehall.content_store.content_item('/')
+    level_one_taxon_hashes = root_taxon.dig('links', 'level_one_taxons') || []
+    taxons = level_one_taxon_hashes.map do |level_one_taxon_hash|
+      Taxonomy::Taxon.from_taxon_hash(level_one_taxon_hash)
+    end
+    live_taxons = taxons.select { |t| t.phase == 'live' }
+    live_taxons.sort_by(&:name)
+  rescue GdsApi::ContentStore::ItemNotFound
+    {}
+  end
+end
```

Comment:
Could we cache this, with some sensible timeout (e.g. 5 to 10 minutes)? Just to avoid too many unnecessary requests?

Reply:
done

Reply:
:+1: